### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by ridabensalem**
### Portfolio desktop version
in this version we : 

- created portfolio for desktop screens 
- added media queries to make the design responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,310 @@
...
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button class="button" type="button">See Project</button>
+        </div>
```
Comment (Yazino12) [inline index.html]: - [ ] To replicate the given template design as much as possible and to an overall better UI 👉🏽 Kindly try to match the given template when it comes to positioning and the space between the elements 👇🏽.

### Your design
image

### Figma template
image

File: index.html [hunk 2]
```diff
@@ -4,10 +4,310 @@
...
+            <ul>
+                <li class="down"><img src="./image/imgg1.png" alt="line2"></li>
+            </ul>
+        </div>
+            </div>
+        </section>
+        </div>
+            <div class="form-contact">
+                 <center> <div class="info-form">
+                    <h2 class="h002">Contact me</h2>
```
Comment (Yazino12) [inline index.html]: - [ ] Kindly consider centering your form title to be consistent with the design throughout your portfolio 👍🏽

### Your design
image

### Figma template
image

File: style.css [hunk 3]
```diff
@@ -1,3 +1,1076 @@
...
+}
+
+@media only screen and (max-width: 768px) {
+  .main {
+    background-image: url("./image/body.png");
+    background-size: 100%;
+    background-position-y: 3rem;
+    height: auto;
+    background-repeat: no-repeat;
+  }
```
Comment (Yazino12) [inline style.css]: - [ ] Even though you did an amazing job designing the Desktop version 🙌🏽, however, you also need to consider the mobile version to keep the page responsive. The current **Media Query** you have in your **style.css** will only apply to screen sizes of size `768px` and larger, this means that your styles will only apply to larger screen sizes, and that's why your page doesn't look as expected on mobile screens 🤔. Please consider adding the styles for your mobile screen sizes 👇🏽

image
image
image
image

**Review by Yazino12 — CHANGES_REQUESTED** [review]
## Hi @ridabensalem 🖐🏾,

You did well implementing some of the project requirements 🎯, but there are still some changes that you need to work on before moving to the next project 🚀🚀.

### To Highlight:
- Correct use of GitHub flow ✔
- Descriptive pull request title and summary ✔

# STATUS: REQUIRED CHANGES ♻♻♻
THERE
_Check the comments under the review 👇🏽👇🏽👇🏽._

- [x] Make sure you update your readme so you are using the latest [Microverse Readme Template](https://github.com/microverseinc/readme-template) and remember that all fields are mandatory except the **FAQ** section 👍🏽. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (ridabensalem) [reply style.css]: I have decided that the hole project need change so I have changed everything even the structure and the classes

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @ridabensalem 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Correct use of GitHub flow ✔
 - Descriptive pull request title and summary ✔

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kingno3l** [comment]
Hi @ridabensalem , 

# Required Changes ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good implementation of linters. No linter errors ✔️
- Descriptive PR 💯 
- Correct Github flow ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kingno3l** [comment]
Name: **Rida** at the top of the page is not aligned properly.

File: index.html (hunk 1 above)
Comment (KAYEMBAJONATHAN) [reply index.html]: - I didn't see anything to comment on but just improve with the structure always try to format your code for it to look clean

**Pull request merged**